examples/windowsservice.xsd

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0"?>
```

Comment:
No longer needed?

Reply:
@oleg-nenashev  What sort of details should be added in `xmlConfigFile.md`? I think adding whole schema file will be an overkill. Will just specifying that we are using embedded XSD schema file to validate XML config file and user can follow that schema file be enough ?

Reply:
See another comment for the response. This comment stands, I am not sure why this additional XSD is needed

Reply:
ping @buddhikac96